sig-cluster-lifecycle/annual-report-2022.md

```diff
@@ -4,64 +4,93 @@
+   tools from k8s.gcr.io to registry.k8s.io.
+   - kubeadm initiated work on migrating its etcd bootstrapping to use ["learner mode"](https://github.com/kubernetes/enhancements/tree/master/keps/sig-cluster-lifecycle/kubeadm/3614-etcd-learner-mode)
+   - We finilized migrating subprojects away from the legacy ["master" label/taint](https://github.com/kubernetes/enhancements/tree/master/keps/sig-cluster-lifecycle/kubeadm/2067-rename-master-label-taint)
+   - We completed the setup of the ["kubernetes-sigs/cluster-api-addon-provider-helm"](https://github.com/kubernetes-sigs/cluster-api-addon-provider-helm/blob/main/OWNERS) subproject providing an option for Cluster API users to manage addons, and most specifically CPI being moved out of tree.
+   - We created the ["logical-cluster"](https://github.com/kubernetes-sigs/logical-cluster/blob/main/OWNERS) subproject to help in cases of CAPI cluster fleets.
+   - We worked tighly with test-infra supporting the migration to registry.k8s.io as well as reviewing test footprint of Cluster API.
+   - We started a Cluster API release team modeled from the Kubernetes release team, and this allowed the introduction of a quarterly release cadence for the entire 2023.
+  
 2. What initiatives are you working on that aren't being tracked in KEPs?
 
-   -
-   -
-   -
-
-
+   - SIG Cluster Lifecyle has not added any new KEPs that affect the whole group or the wider Kubernetes.
+   New KEPs and KEP updates were only done for [kubeadm](https://git.k8s.io/enhancements/keps/sig-cluster-lifecycle/kubeadm),
+   which is the only subproject that is part of the Kubernetes release artifacts.
+   Most of the activity in this group happens outside of the KEP process.
+   Individual projects have implemented their own proposal tracking means, such
+   as the [Cluster API CAEP process](https://sigs.k8s.io/cluster-api/docs/proposals).
+   - A general theme across subprojects is driving them to maturity.
+   With the exception of projects like kubeadm, minikube, kOps and kubespray most of
+   our other subprojects are fairly new in Kubernetes years.
 
 3. KEP work in 2022 (v1.24, v1.25, v1.26):
 
+As noted above, only KEP work for kubeadm was done in 2021.
+Some KEPs were retroactively updated to GA, since the work there was done without KEP updates.
 
 ## Project health
 
 1. What areas and/or subprojects does your group need the most help with?
    Any areas with 2 or fewer OWNERs? (link to more details)
 
-   -
-   -
-   -
+   - ["kubernetes-sigs/cluster-api-addon-provider-helm"](https://github.com/kubernetes-sigs/cluster-api-addon-provider-helm/blob/main/OWNERS)
+   - [etcdadm](https://github.com/kubernetes-sigs/etcdadm)
+   - [cluster-addons](https://github.com/kubernetes-sigs/cluster-addons)
 
 2. What metrics/community health stats does your group care about and/or measure?
 
-   -
-   -
-   -
+   - We started collecting metrics from all of our subprojects using an
```

Comment:
> We did not do a survey for 2022.

Should we?

Reply:
We are late at this point, the deadline is this Friday.
But for sure it makes sense to do it sometime later this year

Reply:
yes let's discuss when we get a chance, I'm happy to help with this